outbox/order-service-quarkus/src/main/java/io/debezium/examples/outbox/order/event/OrderLineUpdatedEvent.java

```diff
@@ -80,4 +60,9 @@ public JsonNode getPayload() {
     public String getType() {
         return "OrderLineUpdated";
     }
+
+    @Override
+    public Long getTimestamp() {
```

Comment:
How about using `Instant` instead (applies more to the interface definition).

Reply:
Done, updated the extension PR and this in commit 7122617.